sudo/lib/log/mod.rs

```diff
@@ -45,6 +74,21 @@ impl SudoLogger {
             .build();
 
         logger.add_logger("sudo::user", stderr_logger);
+        #[cfg(feature = "dev")]
+        {
+            let path = option_env!("SUDO_DEV_LOGS")
+                .map(|s| s.into())
+                .unwrap_or_else(|| std::env::temp_dir().join("sudo-dev.log"));
```

Comment:
maybe this could include the PID and/or timestamp in the name so that multiple sudo invocations don't overwrite the last file